### PR TITLE
Economy Tweaks

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Moves the Civilian account over to the Club Manager
Adds a departmental fund to the civilian account
Increases base pay for the Union account
Adds a small wage for club workers.

## Why It's Good For The Game

Club workers simply put, can't really sustain themselves due to low population
A bit more roundstart humph for the Union
Fixed a major bug with how reagents interacted with exports.
Rebalanced ethanol prices.

## Changelog
```changelog
balance: added base pay for club workers
balance: increased starting union account value
balance: adjusted booze export values to be somewhat in line with prices IRL. Booze is valued between 100-400cr a bottle
fix: fixed reagent calculations for exports.
tweak: club manager now can access the civilian account
tweak: civilian account now has funds
```

### CODE REVIEW

**Comment by Jamini** [comment]
STILL IN PROGRESS.  DO NOT MERGE YET.

File: zzz_modular_syzygy/code/game/jobs/SZdepartment.dm [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/datum/department/guild
+	..()
```
Comment (Michiyamenotehifunana) [inline zzz_modular_syzygy/code/game/jobs/SZdepartment.dm:2]: All child objects of datums inherit from their parents, so putting a callback here is unnecessary. Furthermore, the ..() proc only functions within procs anyways.

File: zzz_modular_syzygy/code/game/jobs/SZdepartment.dm [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	account_initial_balance = 2000 // Less vital than most departments and smaller
+	account_budget = 1000 //A small stipend that the manager can distribute as he sees fit.
+	/*
+		This account pays out to club workers, hydroponics, and custodians.
+		It is managed by The Club manager.
+
+	*/
+
+/datum/job/clubworker
+	..()
```
Comment (Michiyamenotehifunana) [inline zzz_modular_syzygy/code/game/jobs/SZdepartment.dm:17]: Ditto.

File: zzz_modular_syzygy/code/game/jobs/SZdepartment.dm [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	wage = WAGE_LABOUR_DUMB //Club workers make less than professional wages and are expected to make up the difference in tips.
+
+/datum/job/clubmanager
+	..()
```
Comment (Michiyamenotehifunana) [inline zzz_modular_syzygy/code/game/jobs/SZdepartment.dm:21]: Ditto.

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
Honestly surprised that BYOND handled the extraneous callbacks gracefully and managed to compile. Just remove the `..()` from the datums and it'll be all clear.

File: zzz_modular_syzygy/code/game/jobs/SZdepartment.dm (hunk 1 above)
Comment (Jamini) [reply zzz_modular_syzygy/code/game/jobs/SZdepartment.dm]: Done

File: zzz_modular_syzygy/code/game/jobs/SZdepartment.dm (hunk 2 above)
Comment (Jamini) [reply zzz_modular_syzygy/code/game/jobs/SZdepartment.dm]: Done

File: zzz_modular_syzygy/code/game/jobs/SZdepartment.dm (hunk 3 above)
Comment (Jamini) [reply zzz_modular_syzygy/code/game/jobs/SZdepartment.dm]: Done

**Comment by Jamini** [comment]
Completed.

**Review by Michiyamenotehifunana — APPROVED** [review]
All good!

**Pull request merged**